### DISCUSSION TITLE
New listings for sellers using the 'Etsy Marketing/Ads' option issue

### DISCUSSION THREAD

**Discussion by scuculeac-plenty** [etsy/open-api#888] [Q&A]
**Hello,**

we currently have a problem with new listings for sellers using the 'Etsy Marketing/Ads' option.

In the API documentation, the only information we found about this is the "**featured_rank**" flag. 
The documentation notes the following about this:
'The positive non-zero numeric position in the featured listings of the shop, with rank 1 listings appearing in the left-most position in featured listing on a shop’s home page.'

We have run several tests on this, but we can't come to any logical conclusion.

In the first tests we updated three listings with the following values:

**Test 1** 
1412696960: request data -> "featured_rank": 1  / Etsy Ads option: 'Maybe later'
1413426470: request data -> "featured_rank": -1 / Etsy Ads option: 'Maybe later'
1427661513: request data -> "featured_rank": 0  / Etsy Ads option: 'Yes, advertise this listing'

**Test 2**
1412696960: request data -> "featured_rank": 0  / Etsy Ads option: 'Maybe later'
1413426470: request data -> "featured_rank": -1 / Etsy Ads option: 'Maybe later'
1427661513: request data -> "featured_rank": 0  / Etsy Ads option: 'Yes, advertise this listing' (but Listing data shows "featured_rank: 1", even if we send 0)

**Can you provide more information on this?**
a) What value is expected so that the listing is not shown as advertised?
b) What value is expected for the listing to be displayed as advertised?
c) If a value ('positive non-zero numeric position') is submitted - can each number be submitted only once or can multiple listings have the same number?

Also, in our testing we found differences from the submitted values to the values stored on the listing at Etsy.
If we run a 'getListing' call via the V2 API and call the data for a featured listing, we get the value "featured_rank": 2 back there. 
If I call the same data via the 'getListing' call via V3 API, I get "featured_rank": 1.
[Test 1.zip](https://github.com/etsy/open-api/files/10850597/Test.1.zip)
[Test2.zip](https://github.com/etsy/open-api/files/10850598/Test2.zip)

**Comment by turboturtleD** [comment]
Perhaps this won’t help much with what you are TRYING to do, but at least help understand the meaning of featured rank.

Featured rank is a value assigned to listings to be showcased at the top of each shops’ page.  This isn’t as much advertising as it is internal shop arrangement and presentation.  I think you can have up to 5 featured items shown in your shop at any given time, but rank could be assigned beyond 5 to showcase additional items if any of the top 5 ranked items were to sell out, subsequently shifting the rank 6 item to be included in the featured showcase spot at the head of the shop.
